### DISCUSSION TITLE
Localisation

### DISCUSSION THREAD

**Discussion by pejantantangguh** [vercel/next.js#16543] [Help]
Hi all,
Is there any sample code sample when they use next-js in the case below?
Eg. 
When someone from US visit, the URL will be:
example.com/us
from EU URL will be :
example.com/eu

Thank you.

**Comment by Sendouc** [comment]
I believe this example shows how you can achieve this: https://github.com/vercel/next.js/tree/canary/examples/with-i18n-rosetta

**Comment by leerob** [comment]
Check out the new [i18n RFC](https://github.com/vercel/next.js/discussions/17078) and give us your feedback! 🚀
